File: 392-is-subsequence/is-subsequence.cpp
class Solution {
public:
    bool isSubsequence(string s, string t) {
        int s1 = s.size();
        int s2 = t.size();
        int i = 0;
        int j = 0;
        while(i<s1 && j<s2)
             {
                if(s[i]==t[j])
                    {
                        i++;
                       
                    }
                 j++;
                
             }
             if(i>=s1)
                   return true;
                return false;
    }
};

File: 392-is-subsequence/is-subsequence.py
class Solution:
    def isSubsequence(self, s: str, t: str) -> bool:
        i = 0
        j = 0
        count = 0
        n = len(s)
        m = len(t)
        while(i<n and j<m):
            if s[i]==t[j]:
                i+=1
                j+=1
                count+=1
            else:
                j+=1
        
        if count==n:
            return True
        else:
            return False



        